wordpress-mobile/WordPress-iOS
# Show updates available to plugins

Adds available updates to the UI. The possibility to actually update a plugin will come in the next PR

![plugin-updates](https://user-images.githubusercontent.com/8739/33763275-286c211a-dc10-11e7-82dc-a63ddced4fde.png)

See #7471

To test:

- Make sure you have updates available for a plugin
- Go to the plugins list, the update icon should show
- Go to the plugin detail, it should say what version is available
